arlo-client/src/components/Audit/Progress/JurisdictionDetail.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Classes, Dialog } from '@blueprintjs/core'
+import FormButton from '../../Atoms/Form/FormButton'
+import { IJurisdiction } from '../useJurisdictions'
+
+export interface IDialogExampleState {
+  autoFocus: boolean
+  canEscapeKeyClose: boolean
+  canOutsideClickClose: boolean
+  enforceFocus: boolean
+  isOpen: boolean
+  usePortal: boolean
+}
+
+const JurisdictionDetail = ({
+  isOpen,
+  handleClose,
+  jurisdiction,
+  electionId,
+}: {
+  isOpen: boolean
+  handleClose: () => void
+  jurisdiction: IJurisdiction
+  electionId: string
+}) => {
+  if (!jurisdiction) return null // component still loading data
```

Comment:
The type of `jurisdiction` says it won't be falsy, so I think this is unnecessary

Reply:
@jonahkagan I put it in because it errored on initial load. I think it's because I'm setting the initial value of the state hook to the 0th jurisdiction, which is undefined before `useJurisdictions` gets them, so it errors here. This looks to me like the most elegant solution, though I'm open to suggestions.